### DISCUSSION TITLE
Make sendDoorbellMotionNotificationsToTv a switch similar to "In-Home Doorbell"

### DISCUSSION THREAD

**Discussion by shawn3507** [dgreif/ring#651] [Ideas]
It would be nice to have this as a switch. Then it can be turned on and off easily if I am expecting a package.

**Comment by dgreif** [comment]
From what I've heard, motion activated live streams should be part of the iOS 15 release.  When that feature comes out, I'll be removing the sendDoorbellMotionNotificationsToTv feature.  This feature was a bit of a hack, and not something I plan to support long term.
